packages/bpk-component-bottom-sheet/src/BpkBottomSheet.tsx

```diff
@@ -15,86 +15,145 @@
+  isOpen: boolean;
 }
 
 const BpkBottomSheet = ({
   actionText = '',
+  ariaLabelledby,
+  children,
   closeLabel = '',
   closeOnEscPressed = false,
   closeOnScrimClick = false,
   id,
   isOpen,
   onAction = () => null,
   onClose,
-  renderTarget,
   title = '',
-  wide = false,
-  ...rest
+  wide = false
 }: Props) => {
-  const [ exiting, setExitting ] = useState(false);
+  const [exiting, setExiting] = useState(false);
+
+  const animationTimeout = 240;
+
   const handleClose = (
-    arg0?: TouchEvent | MouseEvent | KeyboardEvent,
+    timeout: number,
+    arg0?: TouchEvent | MouseEvent | KeyboardEvent | SyntheticEvent<HTMLDialogElement, Event>,
     arg1?: {
       source: 'ESCAPE' | 'DOCUMENT_CLICK';
     },
-) => {
-    setExitting(true)
+  ) => {
+    setExiting(true)
     setTimeout(() => {
       onClose(arg0, arg1)
-      setExitting(false)
-    }, 240)
+      setExiting(false)
+    }, timeout)
   }
-  return  <Portal
+
+  const headingId = `bpk-bottom-sheet-heading-${id}`;
+  const dialogClassName = getClassName(
+    'bpk-bottom-sheet',
+    wide && 'bpk-bottom-sheet--wide'
+    );
+
+    return <BpkBreakpoint query={BREAKPOINTS.ABOVE_MOBILE}>
+      {(isAboveMobile: boolean) =>
+      <BpkDialogWrapper
+      ariaLabelledby={ariaLabelledby}
+      dialogClassName={dialogClassName}
+      id={id}
       isOpen={isOpen}
-      onClose={handleClose}
+      onClose={(
+        arg0?: TouchEvent | MouseEvent | KeyboardEvent | SyntheticEvent<HTMLDialogElement, Event>,
+        arg1?: {
+          source: 'ESCAPE' | 'DOCUMENT_CLICK';
+        }) => handleClose( isAboveMobile ? 0 : animationTimeout, arg0, arg1)}
+      exiting={exiting}
+      transitionClassNames={{
+        appear: getClassName('bpk-bottom-sheet--appear'),
+        appearActive: getClassName('bpk-bottom-sheet--appear-active'),
+        exit: getClassName('bpk-bottom-sheet--exit')
+      }}
       closeOnEscPressed={closeOnEscPressed}
-      renderTarget={renderTarget}
+      closeOnScrimClick={closeOnScrimClick}
+      timeout={{appear: animationTimeout, exit: isAboveMobile ? 0 : animationTimeout}}
       >
-      <ScrimBpkBottomSheetInner
-        id={id}
-        onClose={handleClose}
-        closeOnScrimClick={closeOnScrimClick}
-        containerClassName={getClassName('bpk-bottom-sheet--container')}
-        title={title}
-        closeLabel={closeLabel}
-        actionText={actionText}
-        onAction={onAction}
-        wide={wide}
-        exiting={exiting}
-        {...rest}
-      />
-    </Portal>
+      <>
+        <header className={getClassName('bpk-bottom-sheet--header')}>
+          <BpkNavigationBar
+            id={headingId}
+            className={getClassName('bpk-bottom-sheet--navigation')}
+            title={title &&
+              <h2
+                id={headingId}
+                className={getClassName('bpk-bottom-sheet--heading')}
+              >
+                {title}
+              </h2>
```

Comment:
This could be used to be a `BpkText`?

```suggestion
              <BpkText id={headingId} textStyle={TEXT_STYLES.label1} tagName="h2">{title}</BpkText>
```